views/application-overview.njk

```diff
@@ -0,0 +1,46 @@
+{% extends 'layout.njk' %}
+{% block content %}
+    <div class="govuk-width-container">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <h2 class="govuk-heading-l">{{name}}</h2>
+                </div>
+            </div>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
```

Comment:
Why 2/3 divs? That would not render the page properly. Instead the second 2/3 div is being rendered after the right column.

The layout should be just one 2/3 div and one 1/3 div. Each div with its content.